go.mod

```diff
@@ -1,22 +1,32 @@
 module github.com/YaleSpinup/spot-api
 
-go 1.16
+go 1.18
 
 require (
 	github.com/YaleSpinup/apierror v0.1.0
-	github.com/felixge/httpsnoop v1.0.2 // indirect
-	github.com/golang/protobuf v1.5.2 // indirect
-	github.com/gopherjs/gopherjs v0.0.0-20210202160940-bed99a852dfe // indirect
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.8.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.10.0
-	github.com/prometheus/common v0.24.0 // indirect
 	github.com/sirupsen/logrus v1.8.1
-	github.com/smartystreets/assertions v1.2.0 // indirect
 	github.com/spotinst/spotinst-sdk-go v1.87.0
```

Comment:
hmm, there should be a newer version of this package
did you run `go get -u`?
also run `go mod tidy`

Reply:
done.